WooCommerce/Classes/POS/Presentation/Barcode Scanner Setup/PointOfSaleBarcodeScannerSetupModels.swift

```diff
@@ -8,32 +8,32 @@ struct PointOfSaleBarcodeScannerSetupFlowOption: Identifiable {
 }
 
 enum PointOfSaleBarcodeScannerType {
-    case socketS720
     case starBSH20B
     case tera12002D
+    case netum1228BC
     case other
 
     var name: String {
         switch self {
-        case .socketS720:
-            return Localization.socketS720Name
         case .starBSH20B:
             return Localization.starBsh20BName
         case .tera12002D:
             return Localization.tera12002DName
+        case .netum1228BC:
+            return Localization.netum1228BCName
         case .other:
             return Localization.otherName
         }
     }
 
     var analyticsName: String {
         switch self {
-        case .socketS720:
-            return "Socket_S720"
         case .starBSH20B:
             return "Star_BSH_20B"
         case .tera12002D:
             return "Tera_1200_2D"
+        case .netum1228BC:
+            return "netum_1228bc"
```

Comment:
super nit: naming consistency
```suggestion
            return "Netum_1228BC"
```